src/pretix/control/views/typeahead.py

```diff
@@ -632,10 +632,9 @@ def meta_values(request):
 
         defaults = defaults.filter(organizer_id=organizer.pk)
         matches = matches.filter(event__organizer_id=organizer.pk)
-        all_access = (
-            (request and request.user.has_active_staff_session(request.session.session_key))
-            or request.user.teams.filter(all_events=True, organizer=organizer).exists()
-        )
+        all_access = request.user.has_active_staff_session(
+            request.session.session_key) if request else request.user.teams.filter(
+            all_events=True, organizer=organizer).exists()
```

Comment:
After we figured out the other one caused a problem I'd very much prefer to not have this modification as well. It's not the same thing.